src/libs/SearchUIUtils.ts

```diff
@@ -489,8 +489,11 @@ function getReportSections(data: OnyxTypes.SearchResults['data'], metadata: Onyx
                 from: data.personalDetailsList?.[reportItem.accountID ?? CONST.DEFAULT_NUMBER_ID],
                 to: reportItem.managerID ? data.personalDetailsList?.[reportItem.managerID] : emptyPersonalDetails,
                 transactions,
-                reportName: isIOUReport ? getIOUReportName(data, reportItem) : reportItem.reportName,
```

Comment:
To confirm @nkdengineer if the expense report uses a report formula we will use that report name?